packages/fx-core/src/common/azure-hosting/azureService.ts

```diff
@@ -77,8 +76,27 @@ export abstract class AzureHosting {
   }
 
   async updateBicep(bicepContext: BicepContext, pluginId: string): Promise<ResourceTemplate> {
-    return {} as ArmTemplateResult;
+    // * The order matters.
+    // * 0: Configuration Orchestration, 1: Configuration Module
+    if (!this.configurable) {
+      return {} as ResourceTemplate;
+    }
+    const bicepFile = `${this.hostType}Configuration.template.bicep`;
+
+    const bicepTemplateDir = this.getBicepTemplateFolder();
+    let module = await generateBicepFromFile(path.join(bicepTemplateDir, bicepFile), bicepContext);
+    module = AzureService.replacePluginId(module, pluginId);
+
+    return {
+      Configuration: this.configurable
```

Comment:
Since we have already return when this.configurable === false in L81, do we still need ? : here?